resources/js/Pages/Results.vue

```diff
@@ -398,6 +398,10 @@ function _handleConfirmation(){
     margin-bottom: var(--dimension-layout-xsmall);
 }
 
+#instructions-button {
+    flex-shrink: 0;
```

Comment:
is there a reason why this is not being set for the whole app? instead of individually for Home and Results page? or in other words... which instances of `cdx-button` need the ellipsis?

Reply:
I see the design now. So only one button should have an ellipsis? :thinking: 
![Uploading image.png…]()

Reply:
I was debating this exact point myself. I decided to play it safe and go with every button that has a heading next to it, where we originally added the gap between button and heading text ("More information" button, "Random Mismatches" and the instructions button). 

I'm not against adding this globally, i just wasn't sure about possible side effects

Reply:
This seems a perfect occasion to ask @sai-san